packages/neon-ledger/src/main.ts

```diff
@@ -90,26 +137,32 @@ export async function getPublicKey(
 /**
  * Requests the device to sign a message using the NEO application.
  * @param ledger - Ledger instance
- * @param hex - message to sign as a hexstring.
+ * @param payload - message to sign as a hexstring.
  * @param bip44String - BIP44 string (40 bytes)
+ * @param networkMagic - MainNet, TestNet or custom network magic
```

Comment:
I thought you are going to pass a string in when you mention MainNet or TestNet. I thought we moved away from the word 'magic'?

Reply:
> I thought we moved away from the word 'magic'?

Did we? I know on the C# side they now call it `network` instead of `magic`, but on the `neon-js` a search still shows a lot of results for `magic`. Should I call it `network`?

Reply:
yeah lets use `network`, gonna change it when I get some time.